resources/views/dashboard/controllers/bookings.blade.php

```diff
@@ -0,0 +1,110 @@
+    <div class="col-lg-6">
+        @if(count($bookings) == 0)
+            <div class="card mb-3">
+                <div class="card-body text-center">
+                    <h4>No ATC Bookings</h4>
+                    <p>There are no ATC Bookings at this time. Create the first one using the form to the right!</p>
+                    <h4 class="fa fa-clock"></h4>
+                </div>
+            </div>
+        @else
+            @foreach($bookings as $date => $date_bookings)
+                <table class="table">
+                    <thead class="thead-light">
+                        <tr>
+                            <th colspan="2">{{ $date }}</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        @foreach($date_bookings as $b)
+                            <tr>
+                                <td>
+                                    <strong>{{ $b->callsign }}</strong>
+                                    &emsp;
+                                    <span>{{ $b->start_time_formatted }} - {{ $b->end_time_formatted }}</span>
+                                    @if($b->cid == Auth::id() || Auth::user()->can('snrStaff'))
+                                        <span class="float-right"><a href="/dashboard/controllers/bookings/delete/{{ $b->id }}" class="btn btn-sm btn-danger">Cancel</a></span>
+                                    @endif
+                                </td>
+                            </tr>
+                        @endforeach
+                    </tbody>
+                </table>
+            @endforeach
+        @endif
+        <small class="d-block mb-3"><i>Only bookings you create are visible beyond 14 days from today.</i></small>
+        <small class="d-block"><i>Bookings allow pilots to plan flights within ATC coverage. This system does not reserve a controller position for a single person. Please refer to the facility general policy for more information.</i></small>
+    </div>
+    <div class="col-lg-6">
+        <div class="card">
+            <div class="card-header">
+                <h5>New Booking</h5>
+            </div>
+            <div class="card-body">
+                {!! Form::open(['action' => 'AtcBookingController@createBooking']) !!}
+                    @csrf
+                    <div class="form-group">
+                        <div class="row">
+                            <div class="col-md-6">
+                                {!! Form::label('facility', '3-Letter Facility Identifier') !!}
+                                {!! Form::text('facility', null, ['placeholder' => 'i.e. ATL, CLT, ZTL', 'class' => 'form-control']) !!}
+                            </div>
+                            <div class="col-md-6">
+                                {!! Form::label('position', 'Position') !!}
+                                {!! Form::text('position', null, ['placeholder' => 'i.e. TWR, APP, CTR', 'class' => 'form-control']) !!}
```

Comment:
Please add a note here reflecting the following policy: "Bookings are limited by ZTL policy to TWR positions and above."

Reply:
Resolved and squashed into b265bba